package.json

```diff
@@ -21,7 +21,7 @@
     "type": "git",
     "url": "git+https://github.com/olivierlsc/swagger-specification-ts.git"
   },
-  "author": "Olivier LIN-SI-CHENG",
+  "author": "Alexander Bauer",
```

Comment:
really?

Reply:
Relax, you can override that. It is just for local assignment in our team, so ppl know whom to talk to internally. Also this was originally not part of the PR.

Reply:
okay, did you look at the fork i already made? I fixed many things and i have prepared 2.0.0 in progress which will completely cover whole functionality swagger 2. After that i will continue to support of openapi 3 functionalities.

Reply:
> Relax, you can override that. It is just for local assignment in our team, so ppl know whom to talk to internally. Also this was originally not part of the PR.

But why do you then out it into the PR?